Always build stdbuf irrespective of whether this is an ELF system.

--- coreutils-8.16/configure.ac.orig	2014-03-18 20:59:59.283886213 -0700
+++ coreutils-8.16/configure.ac	2014-03-18 21:02:30.881534195 -0700
@@ -395,16 +395,13 @@
   gl_ADD_PROG([optional_bin_progs], [df])
 fi
 
-# Limit stdbuf to ELF systems with GCC
 AC_MSG_CHECKING([whether this is an ELF system])
 AC_EGREP_CPP([yes], [#if __ELF__
 yes
 #endif], [elf_sys=yes], [elf_sys=no])
 AC_MSG_RESULT([$elf_sys])
-if test "$elf_sys" = "yes" && \
-   test "$GCC" = "yes"; then
-  gl_ADD_PROG([optional_bin_progs], [stdbuf])
-fi
+gl_ADD_PROG([optional_bin_progs], [stdbuf])
+gl_ADD_PROG([optional_pkglib_progs], [libstdbuf.so])
 
 ############################################################################
 mk="$srcdir/src/Makefile.am"
